examples/client.go

```diff
@@ -243,19 +243,17 @@ func main() {
 
 	case "startCron":
 		fmt.Println("Starting a Cron job")
-		resp, err := r.StartCronJob(job.JobKey())
+		err := r.StartCronJob(job.JobKey())
```

Comment:
might just have `fatalOnErr(err error)` and become onliner `fatalOnErr(r.StartCronJob(job.JobKey()))`, it makes example shorter and harder to copy and paste, which force user to think twice

````go
func fatalOnErr(err error) {
   if err != nil {
        log.Fatal(err)
   }
}
````